service/administration/user.go

```diff
@@ -0,0 +1,466 @@
+package administration
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/jsonutil"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/uritemplates"
+)
+
+type User struct {
+	Email               *string       `json:"email,omitempty"`
+	FirstName           *string       `json:"firstName,omitempty"`
+	LastName            *string       `json:"lastName,omitempty"`
+	Password            *string       `json:"password,omitempty"`
+	Role                *string       `json:"role,omitempty"`
+	UserID              *string       `json:"userId,omitempty"`
+	Username            *string       `json:"username,omitempty"`
+	Type                *string       `json:"type,omitempty"`
+	Mfa                 *bool         `json:"mfa,omitempty"`
+	Policies            []*UserPolicy `json:"policies,omitempty"`
+	Tokens              []*Token      `json:"tokens,omitempty"`
+	PersonalAccessToken *string       `json:"personalAccessToken,omitempty"`
+	Id                  *int          `json:"id,omitempty"`
```

Comment:
we have few more attributes returned in the create user response

Reply:
added